rules-tests/CodeQuality/Rector/Class_/EventListenerToEventSubscriberRector/Fixture/skip_when_as_event_listener_attribute_exists_on_methods.php.inc

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Rector\Symfony\Tests\CodeQuality\Rector\Class_\EventListenerToEventSubscriberRector\Fixture;
+
+class MultiMethodListener
+{
+    #[AsEventListener]
+    public function onMethodA()
+    {
+    }
+
+    #[AsEventListener]
+    public function onMethodB()
+    {
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Symfony\Tests\CodeQuality\Rector\Class_\EventListenerToEventSubscriberRector\Fixture;
+
+class MultiMethodListener
+{
+    #[AsEventListener]
+    public function onMethodA()
+    {
+    }
+
+    #[AsEventListener]
+    public function onMethodB()
+    {
+    }
+}
+
+?>
```

Comment:
second part is not needed for skip fixture

```suggestion
```

Reply:
I pushed the changes